### ISSUE TITLE
Check crates.io library publishing

### ISSUE THREAD

**Issue by andrewdavidmackenzie** [andrewdavidmackenzie/flow#441]
Check that the package published to crates./io has all the files necessary to be used as a flow-lib if downloaded and installed with cargo

**Comment by andrewdavidmackenzie** [comment]
Download using cargo download:
https://crates.io/crates/cargo-download

```
cargo download -x flowstdlib
cd flowstdlib<version>
cargo build
```

**Comment by andrewdavidmackenzie** [comment]
flow.toml files were excluded from the files published to crates.io. Enabled upload and published 0.34.9

**Comment by andrewdavidmackenzie** [comment]
Fixed by #870 and previous committs to Master
